.editorconfig

```diff
@@ -0,0 +1,11 @@
+[*]
```

Comment:
but is it something that is standard in brooklyncentral?
If not should it be and does it warrant a discussion on the mailing list